### ISSUE TITLE
iOS onAccessibilityAction bug

### ISSUE THREAD

**Issue by alexkramerbanjo** [callstack/react-native-slider#250]
## Environment

info Fetching system and libraries information...
System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz
    Memory: 21.64 GB / 31.72 GB
  Binaries:
    Node: 12.10.0 - ...\AppData\Local\Temp\yarn--1606148773529-0.05497837007190132\node.CMD
    Yarn: 1.22.0 - ////AppData\Local\Temp\yarn--1606148773529-0.05497837007190132\yarn.CMD
    npm: 6.10.3 - ....nodejs\npm.CMD
    Watchman: 4.9.4 - C:\bin\watchman.EXE
  SDKs:
    Android SDK:
      API Levels: 23, 28, 29
      Build Tools: 28.0.3, 29.0.3
      System Images: android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: Version  3.5.0.0 AI-191.8026.42.35.6010548
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: ^0.61.5 => 0.61.5

## Description

When passing onAccessibilityAction to the slider, I see the following error: 
Exception thrown while executing the UI block: 1 [RNCSlider setOnAccessibilityActon:]:
unrecognized selector set to instance 0x7f9179fa4900.  

I'm passing onAccessibilityAction to this component because I'm unable to decrement the slider in accessibility mode in iOS.  

## Reproducible Demo

Here's the change hander: 

const handleAccessibleChangeMaxWaitTime = increment => {
    const times = [5, 15, 30, 45, null];
    let index = times.indexOf(maxWaitTime);
    if (index === -1) index = 4;
    if (increment) {
      if (index === times.length - 1 || !maxWaitTime) return;
      setLocationMaxWaitTime(times[index + 1]);
    } else {
      if (index === 0) return;
      setLocationMaxWaitTime(times[index - 1]);
    }
  };

Here's the onAccessibilityAction props I'm passing:

              accessibilityActions={[
                { name: 'increment' },
                { name: 'decrement' },
              ]}
              onAccessibilityAction={event => {
                switch (event.nativeEvent.actionName) {
                  case 'increment':
                    handleAccessibleChangeMaxWaitTime(true);
                    break;
                  case 'decrement':
                    handleAccessibleChangeMaxWaitTime(false);
                    break;
                  default:
                    break;
                }
              }}

I've observed this problem both on the current release and on the v4.0.0-rc.2.  Works fine on Android.

**Comment by alexkramerbanjo** [comment]
I fixed this issue by wrapping the Slider in a View with the accessibility props on that.  Like so:

```
<View
          accessible
          accessibilityRole="adjustable"
          accessibilityActions={[
            { name: 'increment' },
            { name: 'decrement' },
          ]}
          onAccessibilityAction={event => {
            switch (event.nativeEvent.actionName) {
              case 'increment':
                handleAccessibleChangeMaxWaitTime(true);
                break;
              case 'decrement':
                handleAccessibleChangeMaxWaitTime(false);
                break;
              default:
                break;
            }
>
<Slider {...otherProps}/>
</View>
```

Leaving this issue open because this component indicates that it inherits View props, so this should be unnecessary.

**Comment by marcshilling** [comment]
Thanks, your workaround seems to be the only way to get this working. I tried the props added by this PR https://github.com/callstack/react-native-slider/pull/175, but they don't seem to work at all.

**Comment by BartoszKlonowski** [comment]
@alexkramerbanjo I was able to reproduce this issue on my end and I will work on it.
Thank you for taking the time to report!
